Update readme (#1)
DevExpress-Examples/asp-net-web-forms-grid-sort-columns-containing-integer-and-string-values: Readme.md

<!-- default badges list -->
[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E2958)
[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
<!-- default badges end -->
# Grid View for ASP.NET Web Forms - How to sort a column that contains integer and string values
<!-- run online -->
**[[Run Online]](https://codecentral.devexpress.com/e2958/)**
<!-- run online end -->

> This example demonstrates how to sort the Grid View's column that contains values of integer and string types.

Reviewer: ```suggestion
This example demonstrates how to sort columns in the Grid View that contain integer and string values.
```
